conf-template/starport.conf

```diff
@@ -59,6 +64,16 @@ krux.starport {
         # the metric prefix to be used
         prefix = "stats.prod.starport"
     }
+
+    # cloudwatch integration (optional)
+    metric.cloudwatch {
+
+        environment = ???
+        region = ???
+        key = ???
```

Comment:
why dow e need `environment` or `key`?

Reply:
we should reuse the same region where starport is deployed, so probably don't need a specific `region` configuration for cloudwatch neither

Reply:
`environment` and `key` are just there for reporting clarity - for instance in staging they will report `noted-grubworm-usw2-stg` and `staging`. They get fed into the `GlobalDimensions` for Cloudwatch.

Reply:
OK I have removed these and we will use `krux.starport.prefix` to report the environment as per your other comment. I will leave the `metrics.cloudwatch` blob there though in case we require other information in the future.